lib/BloqadeWaveforms/test/waveform.jl

```diff
@@ -167,6 +167,12 @@ end
     end
 end
 
+begin
+
+
+end
```

Comment:
```suggestion
```
Would remove all the associated whitespace as well between `begin` and `end` here